### DISCUSSION TITLE
トップページのデザインについて

### DISCUSSION THREAD

**Discussion by ENDOTAKUMI** [SymbaX/symbax#41] [Ideas]
#28 トップページの背景とか学校で使うサービスなら
学校内の写真とかでもありかも

**Comment by ENDOTAKUMI** [comment]
@MACHIDAMinoru

**Reply by MACHIDAMinoru** [reply]
学内の写真いいかも。検討してみる。

**Comment by ENDOTAKUMI** [comment]
@MACHIDAMinoru 
順調？

**Reply by MACHIDAMinoru** [reply]
@ENDOTAKUMI 
まぁまぁ、順調かな。調べながらやってみてだから作業遅いけど。現在は@media試してみてる。
